darglint/driver.py

```diff
@@ -50,6 +50,15 @@
         'Specifies functions regex to ignore.'
     ),
 )
+parser.add_argument(
+    '--ignore-raise',
+    '-ir',
```

Comment:
Could we change the short option to be a single letter?  Like `-c` or something?